enginetest/trigger_queries.go

```diff
@@ -1976,6 +1976,86 @@ end;`,
 				Expected: []sql.Row{{0}},
 			},
 		},
+	}, {
+		Name: "non-existent procedure in trigger body",
+		SetUpScript: []string{
+			"CREATE TABLE t0 (id INT PRIMARY KEY AUTO_INCREMENT, v1 INT, v2 TEXT);",
+			"CREATE TABLE t1 (id INT PRIMARY KEY AUTO_INCREMENT, v1 INT, v2 TEXT);",
+			"INSERT INTO t0 VALUES (1, 2, 'abc'), (2, 3, 'def');",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    "SELECT * FROM t0;",
+				Expected: []sql.Row{{1, 2, "abc"}, {2, 3, "def"}},
+			},
+			{
+				Query:    "CREATE TRIGGER trig AFTER INSERT ON t0 FOR EACH ROW BEGIN CALL add_to_t1(NEW.v1, NEW.v2); END;",
+				Expected: []sql.Row{{sql.OkResult{}}},
+			},
+			{
+				Query:    "CREATE PROCEDURE add_to_t1(num INT, msg TEXT) INSERT INTO t1 (v1, v2) VALUES (num+3, msg);",
+				Expected: []sql.Row{{sql.OkResult{}}},
+			},
+			{
+				Query:    "INSERT INTO t0 (v1, v2) VALUES (5, 'ggg');",
+				Expected: []sql.Row{{sql.OkResult{RowsAffected: 1, InsertID: 1}}},
+			},
+			{
+				Query:    "SELECT * FROM t0;",
+				Expected: []sql.Row{{1, 2, "abc"}, {2, 3, "def"}, {3, 5, "ggg"}},
+			},
+			{
+				Query:    "SELECT * FROM t1;",
+				Expected: []sql.Row{{1, 8, "ggg"}},
+			},
+		},
+	}, {
+		Name: "procedure that triggers with non-existent procedure in trigger body",
```

Comment:
Does this test case cover more or different surface area than the previous test case (`non-existent procedure in trigger body`)? It looked very similar to me, and seems like it's testing the same code paths in the same ways, but let me know if I'm missing something. 

If this test case isn't covering more than the previous test case, it would be nice to keep the test cases as simple as possible and take this one out in favor of `non-existent procedure in trigger body`.